cesium_app/tests/frontend/test_tooltips.py

```diff
@@ -11,46 +11,55 @@ def test_tab_tooltips(driver, project):
     hover = ActionChains(driver).move_to_element(
         driver.find_element_by_id('react-tabs-0'))
     hover.perform()
-    time.sleep(0.8)
+    time.sleep(1)
     assert driver.find_element_by_xpath(
         "//span[contains(text(),'Manage your projects')]"
     ).is_displayed()
 
     hover = ActionChains(driver).move_to_element(
         driver.find_element_by_id('react-tabs-2'))
     hover.perform()
-    time.sleep(0.8)
+    time.sleep(1)
     assert driver.find_element_by_xpath(
         "//span[contains(text(),'Upload your time-series data')]"
         ).is_displayed()
 
     hover = ActionChains(driver).move_to_element(
         driver.find_element_by_id('react-tabs-4'))
     hover.perform()
-    time.sleep(0.8)
+    time.sleep(1)
```

Comment:
Should we use our standard approach here for awaiting a new element, instead of doing an explicit sleep?

Reply:
Right, the problem with that approach is that the element exists before it is displayed. Thus the `sleep` here.

Reply:
Perhaps, then, we are checking for the wrong thing.  Should we check that the element is visible/selected/contains the expected text/whatever is applicable in each instance instead?

Reply:
That's effectively what we're doing - checking that the element is displayed and contains the expected text.

Reply:
Ah, but you need to wait until it is updated, otherwise it fails too early.  I was hoping for a construct like "wait until this condition is true; but keep trying for at least 5 seconds".  Does that exist?  I'm merging in the mean time.